config/env_blank.json

```diff
@@ -2,6 +2,8 @@
   "LOG_LEVEL": "DEBUG",
   "CANVAS_ACCOUNT_ID": 1,
   "CANVAS_TERM_ID": 164,
+  "CANVAS_TOKEN":"",
+  "CANVAS_URL": "",
```

Comment:
Since I am using the different library async calling and it is handling all the Http operation so I  need to use direct canvas API;s

Reply:
We should probably talk about whether we can use a mixed approach. I've heard from T&L leadership that personal access tokens are only okay in the short-term.

Reply:
We need to have some sort of admin token created for this. This PR using a different library as http client is a quick whip to reduces the timing on the process. I don't know if we need to think a long time solution

Reply:
I'm just communicating what was told to me. I agree that in the short term the personal access token is an easy solution (it is also for GraphQL). I will say, in terms of what I'm seeing and hearing, this is likely going to be a long-term project.

Reply:
Let's use the token for now. I hope we will see the /audit/account API release available soon. We can switch back to the API directory approach by then.